csrankings-1.csv

```diff
@@ -1678,7 +1678,8 @@ César Augusto Fonticielha De Rose,PUC-RS,http://www.inf.pucrs.br/~derose,MliaRD
 César Augusto Missio Marcon,PUC-RS,http://www.inf.pucrs.br/~marcon,QbIUVcQAAAAJ
 César Sánchez,IMDEA Software Institute,http://software.imdea.org/~cesar,M1shlR0AAAAJ
 César Torres 0001,University of Texas at Arlington,http://cearto.com,9CJbh1MAAAAJ
-Cătălin Hrițcu [SP],Max Planck Society,https://prosecco.gforge.inria.fr/personal/hritcu,d-3XtW8AAAAJ
+Cătălin Hrițcu [SP],Max Planck Society,https://prosecco.gforge.inria.fr/personal/hritcu,d-3XtW8AAAAJ
+Çigdem Aslay,Aarhus University,https://cs.au.dk/~cigdem,aVRi34QAAAAJ

```

Comment:
This homepage is currently a stub, so I can't validate it.

Reply:
Sorry will fix it (had provided my homepage-to-be which I haven't set up yet)